src/components/ActionsDialog.tsx

```diff
@@ -61,12 +61,14 @@ const ActionsDialog = ({
                 <div className="h-[1px] mx-8 bg-[#D8D8D8]" />
                 <div className="py-6 px-8">
                   <div className="flex justify-end gap-6">
-                    <button
-                      className="text-btn-xs flex justify-center rounded  text-[#A3A5B9] border-[#A3A5B9] border-2 px-6 py-3 bg-gray-500 bg-opacity-0 hover:bg-opacity-10 active:bg-opacity-20"
-                      onClick={onSecondaryAction}
-                    >
-                      {secondaryActionLabel}
-                    </button>
+                    {secondaryActionLabel && onSecondaryAction && (
```

Comment:
The type makes them independently optional but this is showing they go together. Consider making the type like `secondaryAction?: [label: string, onPress: () => void]`

Reply:
Done. I was just hesitant about un-flattening the params because it impacts equality checking, but for this it's not much impact.